website/docs/Examples/Video_Experiment.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: VideoExperiment
-title: Puppies and Kittens Experiment
+title: Trigger Addition
```

Comment:
This title is not helpful.

Reply:
@evaesteban wanted me to add this

Reply:
It doesn't make sense and there are other parts of the doc that would need to be changed.

Reply:
Also, is not related to the purpose of this pull request. It's a minor change that decreases clarity.